resqpy/organize.py

```diff
@@ -552,8 +404,6 @@ def create_xml(self, add_as_part = True, add_relationships = True, originator =
             self.model.create_reciprocal_relationship(rfuf, 'destinationObject', top_boundary_root, 'sourceObject')
             self.model.create_reciprocal_relationship(rfuf, 'destinationObject', base_boundary_root, 'sourceObject')
 
-      self.root_node = rfuf
```

Comment:
Here and elsewhere: the root_node attribute is (from calling code perspective) replaced with root. This might break some things, though as we want to move away from calling code working with roots, maybe it's okay for things to break now!

Reply:
root_node is still present as a deprecated alias